README.md

```diff
@@ -1461,4 +1462,4 @@ A registry allows you to publish your Rust libraries as crate packages, to share
 
 ## License
 
-[![CC0](https://licensebuttons.net/p/zero/1.0/88x31.png)](https://creativecommons.org/publicdomain/zero/1.0/)
\ No newline at end of file
+[![CC0](https://licensebuttons.net/p/zero/1.0/88x31.png)](https://creativecommons.org/publicdomain/zero/1.0/)
```

Comment:
Not sure why this line changed 🤔 (i edited it via the github ui)

Reply:
Looks like a whitespace thing. Not really fussed too much thankfully :-)